client/tests/test_source_columns.py

```diff
@@ -0,0 +1,30 @@
+import featureform as ff
+import inspect
+from typing import Union, Optional
+from featureform.register import (
+    SourceRegistrar,
+    SubscriptableTransformation,
+)
+
+
+def test_client_has_columns_method():
+    client = ff.Client()
+    assert hasattr(client, "columns")
+
+
+def test_columns_method_signature():
+    sig = inspect.signature(ff.Client.columns)
+
+    params = sig.parameters
+
+    for name, param in params.items():
```

Comment:
is `params` a dictionary? if yes, can we just do `assert params["source"].annotation == Union[SourceRegistrar, SubscriptableTransformation, str]`